### ISSUE TITLE
Windows-Version available? :-)

### ISSUE THREAD

**Issue by thomasmebes** [UNINETT/fwbuilder#2]
Hello!

Is a windows-execute installation file available? :-) Would be great!

Best regards and thanks,

Thomas

**Comment by sirius** [comment]
Hey, @thomasmebes! Sorry for the delay of almost six months. I've fixed new deployment systems and generated unofficial binary packages for W32 and OSX [here](https://github.com/sirius/fwbuilder/releases/tag/v5.3.6-1-gf09bdd1). I'll check with UNINETT if we can provide some official packages. I will at least sign them.

Testing and feedback is very welcome. :smile:

**Comment by kriz2** [comment]
Do you have another link for those builds or updated instructions for a windows compile? 
I'm getting "configure:error: POSIX threads library not present or not configured" when using the existing README.windows
Thanks

**Comment by sirius** [comment]
I've been testing different toolchains this summer, and I'm landing on MXE for cross-compiling for W32 on Linux.

Can you test this w32 binary, [5.3.7-rc1](https://drive.google.com/folderview?id=0B4V9gxpLkC2AM1M2UDZtdGcySnM), and report back?

**Comment by kriz2** [comment]
So far, so good. I had a small issue: the included plink.exe and pscp.exe didn't work for some reason, but the ones from [Putty's Download Site](http://www.chiark.greenend.org.uk/~sgtatham/putty/download.html) works without problems. I'll post any further issues I find here.

**Comment by sirius** [comment]
Thanks for testing. I'll update to the latest plink and pscp.

**Comment by kriz2** [comment]
I've found another issue. When creating a Subfolder in the Tree, it behaves like an Object Group (is sorted as an object, displays object group editor when double clicked), and crashes the program when I try to delete it. Here is a screenshot, I just created the 'aaa' subfolder and double clicked it:
![subfolder creation issue screenshot](https://cloud.githubusercontent.com/assets/2873242/17550780/072498c6-5ef7-11e6-989a-e06c66e04528.jpg)
If I rename it, the icon changes to an object group icon. It does work right when I create the Subfolder, save and reopen the file. I'm not sure whether this is Windows or version specific, but this does work in the old 5.1 build available from sourceforge.
